src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -141,15 +141,28 @@ function getListBlockType(
   return null;
 }
 
+function getInputTags(
+  config: {
+    inputTags: ?Array<string>,
+    element: ?string,
+  }
+): Array<string> {
+  if (config.inputTags) {
+    return config.inputTags;
+  } else if (config.element) {
+    return [config.element];
+  } else {
+    return [];
+  }
+}
+
 function getBlockMapSupportedTags(
   blockRenderMap: DraftBlockRenderMap
 ): Array<string> {
-  const unstyledElement = blockRenderMap.get('unstyled').element;
   return blockRenderMap
-    .map((config) => config.element)
     .valueSeq()
+    .flatMap((config) => getInputTags(config))
     .toSet()
-    .filter((tag) => tag && tag !== unstyledElement)
```

Comment:
I wasn't sure why we were specifically filtering out `unstyled` blocks. Adding this filter back fixes the failing test, but it breaks the intended functionality of allowing `p` tags to be mapped to an unstyled block.